internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -875,6 +877,25 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context,
 	return clusterAfterUpdate, nil
 }
 
+func updateProxyParams(proxy *hiveext.Proxy, params *models.V2ClusterUpdateParams) bool {
+	update := false
+	if proxy != nil {
+		if proxy.NoProxy != "" {
```

Comment:
should probably be similar to https://github.com/openshift/assisted-service/blob/202ce128d8a31fed5cf1bcfeeb2b3f14176ddab2/internal/controller/controllers/clusterdeployments_controller.go#L740-L745 
if user delete the configuration we should update that as well no? or at least set it to empty string.

Reply:
Updated.
I believe that there a lot of similar issues in the controllers when the user delete/clear configurations.
Worth to check.

Reply:
I saw that, will try to look at it thanks.